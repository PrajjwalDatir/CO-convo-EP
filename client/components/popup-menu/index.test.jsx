import React from 'react';
import { mount } from 'enzyme';
import PopupMenu from './index';

describe('<PopupMenu />', () => {
  it('should match snapshot', () => {
    const wrapper = mount(<PopupMenu />);
    expect(wrapper).toMatchSnapshot();
  });
});
