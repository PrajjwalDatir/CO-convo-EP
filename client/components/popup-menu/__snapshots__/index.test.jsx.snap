// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PopupMenu /> should match snapshot 1`] = `
<PopupMenu
  bgClassName=""
  isOpen={false}
  onClose={[Function]}
>
  <div
    className="invisible"
  >
    <button
      aria-label="Close menu"
      className="fixed inset-0 w-full h-full cursor-default z-40 "
      onClick={[Function]}
      tabIndex="-1"
      type="button"
    />
    <div
      className="relative z-50"
    />
  </div>
</PopupMenu>
`;
